ffc-node-base/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG NODE_VERSION=10.18.0
+FROM node:$NODE_VERSION-alpine
+ARG NODE_VERSION
+ARG VERSION=1.0.0
+
+RUN apk update && apk add --no-cache tini=0.18.0-r0
+ENTRYPOINT ["/sbin/tini", "--"]

```

Comment:
based on @neilbmclaughlin's [comment](https://github.com/DEFRA/ffc-docker-base/pull/1#pullrequestreview-338722399), it may be worth adding a comment here on why we use tini.

Reply:
Comment added